src/components/resources/ResourcesOverview.js

```diff
@@ -1,12 +1,22 @@
 import styled from 'styled-components';
 import {Resource} from './Resource';
+import {nanoid} from '@reduxjs/toolkit';
+import {useSelector} from 'react-redux';
+
+export const ResourcesOverview = () => {
+  const currentUserResources = useSelector(state => state.currentUserResources);
+
+  let currentRess = [];
+
+  Object.keys(currentUserResources).forEach(key => {
+    currentRess.push({name: key, value: currentUserResources[key]});
+  });
 
-export const ResourcesOverview = ({currentResources}) => {
   return (
     <StyledResourcesContainer>
-      {currentResources.map(currentRess => (
-        <Resource key={currentRess.id} displayValue={true} currentRess={currentRess}></Resource>
-      ))}
+      {currentRess.map(ress => {
+        return <Resource key={nanoid()} displayValue={true} currentRess={ress}></Resource>;
```

Comment:
Please don't ever call `nanoid` when passing it as a key! This way the key will always be different and might cause some rendering issues because react doesn't have a way to identify the actual item anymore. I would recommend to add `nanoid` when creating the object for currentRess.
```suggestion
        return <Resource key={ress.id} displayValue={true} currentRess={ress}></Resource>;
```

Reply:
thank you, done